src/core/Word.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "Byte.hpp"
+
+class Word final : public Element
+{
+  public:
+    Word() = default;
+    explicit Word(QString descr, int value);
+    explicit Word(QString descr, int addr, uint8_t low, uint8_t high);
+    ~Word() override = default;
+
+    Word &operator=(const Word &) = default;
+    Word &operator=(Word &&) = default;
+
+    void setValue(int16_t val) override;
+    int value() const override;
+
+    uint8_t low() const noexcept;
+    uint8_t high() const noexcept;
+
+  private:
+    uint8_t m_low{};
+    uint8_t m_high{};
```

Comment:
Since the class provides high and low word parts, I suggest implementing the setters.

Reply:
My suggestion is to implement both getters and setters of high and low part

Reply:
Why? For now who needs setter members? @marsiliano @AndreaRicchi

Reply:
It could be useful for the conversion from the raw value and the real value, and viceversa. e.g: we have a word that represent the status of the probe, and if the first byte it's set to FE or FF to signal that the probe is disconnect or something like that a set or a get could be very useful, instead of creating a method to check the value, also a get / set bit on the whole word could be useful